app/Http/Controllers/CourseUserController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Course;
+use Illuminate\Support\Facades\Auth;
+
+class CourseUserController extends Controller
+{
+    public function store(Request $request)
+    {
+        $course = Course::findOrFail($request['course_id']);
+
+        if (!$course->joined) {
```

Comment:
đặt lại tên attribute đi nhé, joined -> isJoined

với các attribute, biến trả về true/false thì cần có tiền tố  `is` phía trước